client/src/app/components/employees/employee-list/employee-list.component.spec.ts

```diff
@@ -30,4 +33,35 @@ describe('ClientGridViewComponent', () => {
   it('should create Client grid view component', () => {
     expect(component).toBeTruthy();
   });
+
+   // Test that Start Audit button is disabled when no items selected
+  it('Create button should be active', () => {
+    fixture.detectChanges();
+    createBtn = fixture.debugElement.query(By.css('#create')).nativeElement;
+    expect(createBtn.disabled).toBeFalsy();
+  });
+
+  // Test the applyFilter()
+  it('Call method', () => {
+    component.applyFilter('');
+  });
+
+  // Test the populateTable()
+  // it('Call method', () => {
+  //   try {
+  //     component.populateTable([{
+  //       first_name: 'Nick',
+  //       last_name: 'Nick',
+  //       role: 'SA',
+  //       is_active: true,
+  //       id: 2,
+  //       email: 'nick@test.com',
+  //       location: 'Terrebonne',
+  //     }]);
+  //   }
+  //   catch (error) {
+  //     console.error(error);
+  //   }
+  // });
+
```

Comment:
if this isn't needed, can we delete this?

Reply:
As Avery mentioned above, this is commented out is to show others how to setup their tests / serve as an example